modules/cnf-du-creating-a-machine-config-manifest.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-creating-a-machine-config-manifest_{context}"]
+
+= Creating a machine config manifest
+
+To enable workload partitioning, you need to provide a `MachineConfig` manifest during installation to configure CRI-O and kubelet to know about the workload types.
+
+The manifest, without the encoded file content, looks like this:
+
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 02-master-workload-partitioning
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,<01-workload-partitioning-content>
+        mode: 420
+        overwrite: true
+        path: /etc/crio/crio.conf.d/01-workload-partitioning
+        user:
+          name: root
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,<openshift-workload-pinning content>
+        mode: 420
+        overwrite: true
+        path: /etc/kubernetes/openshift-workload-pinning
+        user:
+          name: root
+----
+
+The contents of `/etc/crio/crio.conf.d/01-workload-partitioning` should look like this (the `cpuset` value will vary based on the deployment):
+
+[source,yaml]
+----
+[crio.runtime.workloads.management]
+activation_annotation = "target.workload.openshift.io/management"
+annotation_prefix = "resources.workload.openshift.io"
+resources = { "cpushares" = 0, "cpuset" = "0-1,52-53" }
+----
+
+If hyperthreading is enabled, specify both threads of each core. The cpuset must match the reserved CPU set specified in the PerformanceProfile):
+
+
+This content should be base64 encoded and provided in the `01-workload-partitioning-content` in the manifest above.
+
+The contents of `/etc/kubernetes/openshift-workload-pinning` should look like this (The cpuset must match the value in `/etc/crio/crio.conf.d/01-workload-partitioning`):
```

Comment:
Please avoid the parens.  Please see previous comment related to the ISG.

A callout is a common alternative.  Up to you.